docs/running_manually.md

```diff
@@ -19,10 +19,12 @@ You need:
   - [node v9.11.1](https://nodejs.org/dist/v9.11.1/docs/api/);
   - [npm 5.6.0](https://www.npmjs.com/package/npm/v/5.6.0).
 
-OBS: If you already have another version of node and npm we recommend take a
+If you already have another version of node and npm we recommend take a
 look at the [n package](https://github.com/tj/n) or run Aurelia's docker container as
 presented [here](../run_on_docker.md).
 
+*Note. Recommend to run on  AWS instance `t2.small` or higher.*
```

Comment:
I have a question: If the person is using another cloud provider? I think is better to specify the amount of ram needed instead of the instance type.

Reply:
I just update, what do you think?